packages/block-editor/src/components/dimension-control/README.md

```diff
@@ -0,0 +1,138 @@
+DimensionControl
+=============================
+
+`DimensionControl` is a component designed to provide a UI to control spacing and/or dimensions. It is intended for use within the editor `InspectorControls` sidebar.
+
+## Usage
+
+In a block's `edit` implementation, render a `<DimensionControl />` component. 
+
+
+```jsx
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+import {
+	DimensionControl,
+} from '@wordpress/block-editor';
+
+registerBlockType( 'my-plugin/my-block', {
+	// ...
+
+	attributes: {
+		// other attributes here
+		// ...
+
+		paddingSize: {
+			type: 'string',
+		},
+	},
+
+	edit( { attributes, setAttributes, clientId } ) {
+		
+		const { paddingSize } = attributes;
+		
+
+		const updateSpacing = ( dimension, size, device = '' ) => {
+			setAttributes( {
+				[ `${ dimension }${ device }` ]: size,
+			} );
+		};
+
+		const resetSpacingDimension = ( dimension, device = '' ) => {
+			setAttributes( {
+				[ `${ dimension }${ device }` ]: '',
+			} );
+		};
+
+		return (
+			<DimensionControl
+				label={ __( 'Padding' ) }
+				icon={ 'desktop' }
+				iconLabel={ 'Desktop Devices' }
```

Comment:
Please remove the iconLabel

Reply:
Understood. Thanks for feedback here. I've actioned this.